### PR TITLE
Bump easyscience and easycrystallography to releases and fix minor issues

### PR DESCRIPTION
**Pull request by AndrewSazonov**
* Bump easycrystallography to 0.6.0
* Bump easyscience to 1.1.1
* Add urls to the experimental parameters
* Fix units for parameters dtt1, dtt2, sigma1, sigma2, sigma3
* Clean up project

### CODE REVIEW

File: .github/workflows/test-and-package.yaml [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+    #  with:
+    #    files: ./coverage/coverage.xml
+    #    env_vars: OS,PYTHON
+    #    fail_ci_if_error: true  # optional (default = false)
+    #    name: Pytest coverage
+    #    token: ${{ secrets.CODECOV_TOKEN }}
+    #  env:
+    #    OS: ${{ matrix.os }}
+    #    PYTHON: ${{ matrix.python-version }}
+
```
Comment (rozyczko) [inline .github/workflows/test-and-package.yaml:113]: So the upload doesn't work?

File: README.md [hunk 2]
```diff
@@ -1,140 +1,24 @@
-# [![Unit Tests][20]][21] ![Release][31] [![Downloads][70]][71] [![Lines of code][81]](<>) [![Total lines][80]](<>) [![Files][82]](<>) [![License][50]][51] [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/easyscience/easydiffractionlib/master)
+<img src="https://raw.githubusercontent.com/EasyScience/EasyDiffractionApp/master/resources/images/ed_logo.svg" height="65">
 
-[![](http://github-actions.40ants.com/easyscience/easydiffractionlib/matrix.svg)](https://github.com/easyscience/easydiffractionlib/actions)
+EasyDiffraction is a Python library for the calculation of neutron powder diffraction pattern based on a structural model and refinement of its parameters against experimental data.
```
Comment (rozyczko) [inline README.md:3]: `... for calculating neutron powder...` would probably sound better
also
`... and for refining of its parameters...`

File: pyproject.toml [hunk 3]
```diff
@@ -58,61 +45,66 @@ charts = [
...
 quote-style = 'single'
+indent-style = 'space'  # PEP 8 recommends using spaces over tabs for indentation.
+line-ending = 'lf'  # Line endings will be converted to \n. The default line ending on Unix.
+
+[tool.ruff.lint.isort]
+force-single-line = true
 
 [tool.ruff.lint.per-file-ignores]
-# allow asserts in test files
+# allow asserting in test files
```
Comment (rozyczko) [inline pyproject.toml:89]: `asserts` is a proper wording here.

**Review by rozyczko — APPROVED** [review]
minor wording

File: pyproject.toml (hunk 3 above)
Comment (AndrewSazonov) [reply pyproject.toml:89]: fixed

File: README.md (hunk 2 above)
Comment (AndrewSazonov) [reply README.md:3]: fixed

File: .github/workflows/test-and-package.yaml (hunk 1 above)
Comment (AndrewSazonov) [reply .github/workflows/test-and-package.yaml:113]: Well, this works partially and requires additional tests before enabling this step.

**Pull request merged**